assets/src/edit-story/app/config/context.js

```diff
@@ -3,4 +3,4 @@
  */
 import { createContext } from '@wordpress/element';
 
-export default createContext( { api: {} } );
+export default createContext( { api: {}, statuses: {} } );
```

Comment:
Isn't `statuses` part of `api`?

Reply:
Original post statuses came in via config. I didnt remove this. Removed in https://github.com/ampproject/amp-wp/pull/4122/commits/5f9eb6f5b5dc81a15cc5e571b1f3e6412cd9f2a0